kythe/typescript/testdata/marked_source/function.ts

```diff
@@ -0,0 +1,44 @@
+//- @p1 defines/binding A_p1
+//- A_p1 code A_p1_code
+//- A_p1_code child.0 A_p1_context
```

Comment:
To match the rest of the indexers, the context of a parameter's MarkedSource should be the qualified name of its enclosing function.

Reply:
I see. My impression was the context was the declaration context; for example, for mutable local variable we currently say the context is `let` and for constants it is `const`. For a local variable like `let a = 1` should the context be something else?